requirements.txt

```diff
@@ -1,20 +1,37 @@
 astroid==2.2.5
+beautifulsoup4==4.7.1
+certifi==2019.3.9
+chardet==3.0.4
 coverage==4.5.3
 coveralls==1.7.0
+cssutils==1.0.2
 dj-database-url==0.5.0
 Django==2.1.7
+django-cors==0.1
 django-cors-middleware==1.2.0
 django-decouple==2.1
 django-extensions==2.1.6
+django-inlinecss==0.2.0
+django-naomi==0.8
 django-nose==1.4.6
 djangorestframework==3.9.2
-gunicorn==19.9.0
+docopt==0.6.2
+future==0.17.1
+idna==2.8
 isort==4.3.16
 lazy-object-proxy==1.3.1
 mccabe==0.6.1
+nose==1.3.7
 psycopg2-binary==2.7.7
 PyJWT==1.4.2
+pylint==2.3.1
```

Comment:
This dependency and a couple others in this file should be in dev-requirements.txt not in the requirements.txt